composer.json

```diff
@@ -269,6 +269,8 @@
         ],
         "integration-tests": [
             "@composer create-test-db",
+            "@composer install --working-dir=modules/ps_apiresources",
```

Comment:
there is the dump-autoload command, but dev dependencies are still necessary for the tests to work properly